packages/alea-frontend/pages/api/study-buddy/get-all-users-status.ts

```diff
@@ -0,0 +1,59 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import {
+    executeAndEndSet500OnError,
+    getUserIdOrSetError,
+} from '../comment-utils';
+import { AllCoursesStats, isModerator } from '@stex-react/api';
+export default async function handler(
+    req: NextApiRequest,
+    res: NextApiResponse
+) {
+    const userId = await getUserIdOrSetError(req, res);
+    if (!userId) return;
+
+    if (!isModerator(userId)) {
+        res.status(403).send({ message: 'Unauthorized.' });
+        return;
+    }
+    const result1: any[] = await executeAndEndSet500OnError(
+        `SELECT 
+          COUNT(userId) as TotalUsers, 
```

Comment:
I had these postulates to write this query.

- The sum of all the course's stats should be equal to this number.
- If change the query to remove duplicate people from the count, It is possible to find a person had been registered in many courses. I thought this can break the users' privacy.
- A user always fills all the form input In study body for every course, so technically he/ she is a sprat user.

I am not sure all these postulates are 100% correct. If it is a problem with the postulates, please tell me. 
Also, Where I should write the API documents?

Reply:
>  If it is a problem with the postulates, please tell me. 
It seems fine to me. No need to change
> Where I should write the API documents?
At the top of this function or in study-buddy-api.ts